src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.DataFactory</PackageId>
+    <Description>Azure Data Factory v2 is a fully managed hybrid data service to compose and manage data integration workflows at scale that are reliable and secure. Use native ADF data connectors and Integration Runtimes to move and transform cloud and on-premises data that can be unstructured, semi-structured, and structured with Hadoop, Azure Data Lake, Spark, SQL Server, Cosmos DB and many other data platforms.</Description>
+    <VersionPrefix>0.2.0-preview</VersionPrefix>
+    <AssemblyName>Microsoft.Azure.Management.DataFactory</AssemblyName>
+    <PackageTags>Microsoft Azure resource management;Data Factory;ADF;REST HTTP client;azureofficial;windowsazureofficial;netcore451511</PackageTags>
```

Comment:
Add < PackageReleaseNotes > and add Release notes.
I would recommend to use < ![CDATA and then format your Release notes

https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/MarketplaceOrdering/Management.MarketplaceOrdering/Microsoft.Azure.Management.MarketplaceOrdering.csproj#L10

Reply:
In the past, we have written our own [changelog](https://github.com/Azure/azure-sdk-for-net/blob/master/src/ResourceManagement/DataFactory/DataFactoryManagement/DataFactoryManagementChangelog.md) to serve this purpose and put a link to a [doc page](https://github.com/Azure/azure-sdk-for-net/blob/master/src/ResourceManagement/DataFactory/DataFactoryManagement/Microsoft.Azure.Management.DataFactories.nuspec#L6) that we create with that content. 

Can we do a similar thing this time around? Right now we don't have a link to where that content will be for this new package.